db/changelog/v-1/04-changeset-test-table.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="add-test-table" author="aktsay" context="dev or prod">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists tableName="test"/>
+            </not>
+        </preConditions>
+        <createTable tableName="test">
+            <column name="id" type="varchar(255)">
```

Comment:
I believe we should use bigint for all ids now

Reply:
As I remember this one should be hash from two files.

Reply:
Yes, right